PATTERNS.md

```diff
@@ -1,25 +1,28 @@
 # Patterns in Quamina
 
-**Patterns** are added to Quamina instances using the 
+**Patterns** are added to Quamina instances using the
 `AddPattern` API.  This document specifies the syntax and
```

Comment:
nit: double whitespace

Reply:
All double whitespaces killed in file.  There were lots in the JSON too, no idea why.